### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by ifedapoolarewaju**

### CODE REVIEW

**Comment by kvz** [comment]
Thanks a lot!

File: README.md [hunk 1]
```diff
@@ -129,8 +129,8 @@ response.reload_until_finished! tries: 300 # default is 600
 ```
 
 In general, you use hash accessor syntax to query any direct attribute from
-the [response](http://transloadit.com/docs/assemblies#response-format).
-Methods suffixed by a question mark provide a more readable way of quering
+the [response](https://transloadit.com/docs/#assembly-status-response).
```
Comment (rmehner) [inline README.md]: You've changed this link and it now goes to the doc overview and not to any anchor. Is this what you want?

(Also is true for all the other links you've changed in here)

**Comment by rmehner** [comment]
Hey @ifedapoolarewaju, @kvz was a bit quick to merge, there's a little thing that I'd like you to check out, as it seems wrong :) But no worries, easy to fix.

Anyway, thanks for your continued work on this!

**Comment by kvz** [comment]
Ouch, I think that was my bad, I gave the updated links to Ife and didn't think to double check them 😱 

The correct link I suppose here should become https://transloadit.com/docs/api-docs/#assembly-status-response .

**Comment by ifedapoolarewaju** [comment]
and I didn't double check either.

**Pull request merged**